config/reports.json

```diff
@@ -577,5 +705,25 @@
 				"icon": "fas fa-stopwatch"
 			}
 		}
+	},
+	"progressive-web-apps": {
+		"name": "Progressive Web Apps",
+		"summary": "This report examines the state of Progressive Web Apps (PWA) according to the HTTP Archive.",
```

Comment:
"according to the HTTP Archive" isn't needed :)

You may want to explain what a PWA is and what qualifies a page to be one. Also consider using this summary to explain Lighthouse so you don't need to do it for each metric.

Reply:
Addressed in https://github.com/HTTPArchive/httparchive.org/pull/51/commits/adb69d33d57d3229f19a914c4ef1b4cff4ebdfd3.